net-analyzer/ossec-hids/ossec-hids-3.1.0.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit user
+
+DESCRIPTION="Open Source Host-based Intrusion Detection System"
+HOMEPAGE="https://www.ossec.net/"
+SRC_URI="https://github.com/ossec/ossec-hids/archive/${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="GPL-2"
+SLOT="0"
+
+KEYWORDS="~amd64"
+
+IUSE="agent hybrid local mysql postgres server sqlite zlib"
+DEPEND="mysql? ( virtual/mysql )
+	sqlite? ( dev-db/sqlite:3 )
+	postgres? ( dev-db/postgresql:= )
+	zlib? ( sys-libs/zlib:= )"
+RDEPEND="${DEPEND}"
+
+S="${WORKDIR}/${P}/src"
+
+MY_PREFIX="/var/ossec"
+
+my_options() {
+	local target="local"
+	if use agent; then
+		target="agent"
+	elif use hybrid; then
+		target="hybrid"
+	elif use server; then
+		target="server"
+	fi
+
+	local opts="V=0 USE_SQLITE=$(usex sqlite) ZLIB_SYSTEM=$(usex zlib)"
+	if use mysql; then
+		opts="${opts} DATABASE=mysql"
+	elif use postgres; then
+		opts="${opts} DATABASE=pgsql"
+	fi
+
+	echo ${opts} TARGET=${target}
+}
+
+pkg_setup() {
+	enewuser ossec  -1 -1 ${MY_PREFIX}
+	enewuser ossecm -1 -1 -1 ossec
+	enewuser ossecr -1 -1 -1 ossec
+}
+
+src_prepare() {
+	default
+	eapply "${FILESDIR}/makefile.patch"
+}
+
+src_compile() {
+	emake $(my_options) PREFIX=${MY_PREFIX}
+}
+
+src_install() {
+	keepdir ${MY_PREFIX}/logs/alerts
```

Comment:
You could use `/var/ossec/logs/{alerts,archives,firewall}` etc. to reduce the number of calls.